.eslintrc.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 2018
   },
-  extends: ['eslint-config-uber-es2015', 'prettier'],
+  plugins: ['react'],
+  extends: ['uber-jsx', 'eslint-config-uber-es2015', 'prettier', 'prettier/react'],
```

Comment:
I wonder if we could have those additions in a separate `.eslintrc.js` in `/examples` directory? Since the main `modules` doesn't use React.